sidechain/consensus/aura/src/verifier.rs

```diff
@@ -135,12 +135,18 @@ where
 fn verify_block_ancestry<B: Block>(block: &B, last_block: &B) -> Result<(), ConsensusError> {
 	ensure!(
 		last_block.block_number() + 1 == block.block_number(),
-		ConsensusError::BadSidechainBlock(block.hash(), "Invalid block number".into())
+		ConsensusError::BlockAncestryMismatch(
+			last_block.block_number(),
+			"Invalid block number".into()
+		)
```

Comment:
Ah good catch. But there are other cases then where we return a `BlockAncestryMismatch` error? And those are still correct?

Reply:
Just realized that you added a new enum value - so there were no previous usages of that error type.

Reply:
No, there shouldn't be any, because I newly introduced this Error. (see sidechain/consensus/common/src/error.rs)